test/es-module/test-esm-import-meta.mjs

```diff
@@ -3,20 +3,38 @@
 import '../common';
 import assert from 'assert';
 
+const fixtures = import.meta.require('../common/fixtures');
```

Comment:
this should be exposed by common.mjs

Reply:
I was thinking that using it this way offered an extra smoke test for import.meta.require.

more than happy to change it though

Reply:
at best we can take it as an excuse to build up common.mjs... i don't like the idea of "smoke tests" but that's just an opinion.